### ISSUE TITLE
List of all open ports

### ISSUE THREAD

**Issue by dbuezas** [intx82/a9-v720#19]
Hi, thanks for this incredible piece of smart hack!

I'm making a home assistant add-on and I already managed it to work, but only if I make the docker container run in the host's network.
To keep security in mind, I'd like to expose only the necessary ports.
Using `netstat -a`
I found the python script opens:

- 6123/tcp: 6123 
- 80/tcp: 80 
- 6123/udp: 6123 

but after exposing them in the config.yaml file, the camera is still unable to connect.

Are there any others I should be opening?

**Comment by edouardaihua** [comment]
Hey @dbuezas 
Did you managed to have your HA Add-on work ?
I've been looking at all the good work from this depo but it is a bit too complex for me.

**Comment by dbuezas** [comment]
Yes it's here: https://github.com/dbuezas/ha-addons/tree/main/a9-cam
